kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -33,12 +33,16 @@
       <tbody slot="tbody">
         <tr v-for="(facility, idx) in facilities" :key="idx">
           <td>
-            <FacilityNameAndSyncStatus
-              :facility="facility"
-              :isSyncing="facilityIsSyncing(facility)"
-              :isDeleting="facilityIsDeleting(facility)"
-              :syncHasFailed="facility.syncHasFailed"
-            />
+            <span>
+              <KLabeledIcon icon="facility" style="vertical-align: top; top: 8px" />
```

Comment:
Move the `KLabeledIcon` into the `FacilityNameAndSyncStatus.vue` component so it appears also on the Facility > Data page.

That is, on this line

https://github.com/learningequality/kolibri/blob/8e40f093569e1a6374fc8b536b523aa2fb267716/kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus.vue#L6

Add code like this

```
        <KLabeledIcon icon="facility">
          {{ facility.name }}
        </KLabeledIcon>

```

Notice, that the `facility.name` is placed in the "default" slot of `KLabeledComponent`, so it appears next to the icon.